components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/ServiceBusInitializer.java

```diff
@@ -218,6 +222,20 @@ protected void activate(ComponentContext ctxt) {
                     configurationManager);*/
             // Start Inbound Endpoint Listeners
             EndpointListenerLoader.loadListeners();
+            String injectCarName = System.getProperty(ServiceBusConstants.AUTOMATION_MODE_CAR_NAME_SYSTEM_PROPERTY);
+            if (injectCarName != null && !injectCarName.isEmpty()) {
+                String sequenceName = getMainSequenceName(injectCarName);
+                if (sequenceName == null) {
+                    log.error("Invalid cApp name or main sequence name not found");
```

Comment:
can we break this into two logs for exact reasons?

Reply:
Fixed